models/yang/extensions/openconfig-lldp-ext.yang

```diff
@@ -0,0 +1,25 @@
+module openconfig-lldp-ext {
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://openconfig.net/yang/lldp/extention";
```

Comment:
typo here: 'extention' should be 'extension'

Reply:
Changed it to extension. Based it on interfaces ext yang.